src/code42cli/cmds/profile.py

```diff
@@ -60,6 +60,7 @@ def username_option(required=False):
     is_flag=True,
     help="For development purposes, do not validate the SSL certificates of Code42 servers. "
     "This is not recommended, except for specific scenarios like testing.",
+    default=None,
```

Comment:
When creating a new profile, it still defaults to `false` with this change..
In fact, this is actually a bugfix because it used to override `true`to false if excluded and updating profile.... Migt need to verify that and possibly include in the changelog

Reply:
Yes, I just verified... This a bug but it is not a released bug

Reply:
Reproduce the bug from master branch by doing:

```
code42 profile update --disable-ssl-errors
code42 profile show
# notice you disable-ssl-errors set to True

code42 profile update -u username
# say no to updating password
code42 profile show

# Notice you have disable-ssl-errors set to False
```

Reply:
Actually, it might in sense be in the released version... will verify